mtags/src/main/scala-3/scala/meta/internal/pc/completions/Completions.scala

```diff
@@ -515,11 +515,14 @@ class Completions(
 
               val include =
                 !isUninterestingSymbol(sym) &&
-                  isNotLocalForwardReference(sym)
-                  && isNotAModuleOrModuleIsValidForPos(sym)
-                  && isNotAMethodOrMethodIsValidForPos(sym)
-                  && isNotClassOrTraitOrTheyAreValidForPos(sym)
-
+                  isNotLocalForwardReference(sym) && (
+                    // to not duplicate with package itself
+                    sym.is(Package) ||
+                      isNotPackageObject(sym)
```

Comment:
so if it is a package and is a package object, include should be true?

Reply:
It appears to me that the conditions need to be like this https://github.com/scalameta/metals/pull/4223#discussion_r936947521

Reply:
with `isNotPackageObject` being out and separately "And"ed with all other conditions..

Reply:
I think the result is the same actually, so I will leave it as is.

Reply:
with the way it is written now, if 

- `sym.is(Package)` is `true` and
-  `isNotPackageObject(sym)` is` false`

then `include` would be `true`.

With the suggested way, if  `isNotPackageObject(sym)` is` false`, then `include` is always going to be `false`.

Reply:
Actually this doesn't happen. Both are either true or false.